llvm/utils/TableGen/DAGISelMatcherGen.cpp

```diff
@@ -602,6 +603,7 @@ bool MatcherGen::EmitMatcherCode(unsigned Variant) {
 
     // Emit a CheckComplexPat operation, which does the match (aborting if it
     // fails) and pushes the matched operands onto the recorded nodes list.
+    CGP.increaseComplexPatternUsage(CP);
```

Comment:
Seems unrelated?

Reply:
This is from parent commit, not this PR.